contracts/axelar-soroban-interfaces/src/axelar_operators.rs

```diff
@@ -22,5 +21,15 @@ pub trait AxelarOperatorsInterface {
         contract: Address,
         function_name: Symbol,
         args: Vec<Val>,
-    ) -> Val;
+    ) -> Result<Val, OperatorError>;
+}
+
+#[contracterror]
```

Comment:
How about moving this to the top, before the pub trait interface, for better code readability?

Reply:
Sounds good